### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by lucasmoura**
Fix some integration tests that were affected by the recent cis and messaging changes

## Test Steps
Run the modified integration tests

## Desired commit type::

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: features/attach_validtoken.feature [hunk 1]
```diff
@@ -24,7 +24,6 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
         Then if `<release>` in `focal` and stdout matches regexp:
         """
         \d+ update(s)? can be applied immediately.
-        \d+ of these updates (is a|are) standard security update(s)?.
```
Comment (orndorffgrant) [inline features/attach_validtoken.feature:27]: @lucasmoura I thought we were adding this message back even when there are 0 updates?

Comment (lucasmoura) [reply features/attach_validtoken.feature:27]: No, it is actually the other one we are adding back when we have 0 updates

**Review by orndorffgrant — APPROVED** [review]
LGTM! Once jenkins passes this is good to go

**Comment by lucasmoura** [comment]
CI errors are unrelated to the proposed changes

**Pull request merged**